images/vm-route-forge/internal/server/server.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2024 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"net/http"
+
+	"github.com/emicklei/go-restful/v3"
```

Comment:
Do we need a restful library for a simple healtz? =)))

Reply:
So, maybe you're right, but I think in the future we'll want to add metric handlers, proofs, etc